include/netdb.h

```diff
@@ -222,6 +222,8 @@ struct protoent
                             * alternative protocol names, terminated by a
                             * null pointer. */
   int        p_proto;      /* The protocol number. */
+  uint8_t    idx;          /* Index used by the local database, required
```

Comment:
Let's change `p_proto` to `uint8_t` as well. I do not see that it can carry more than 8-bits

Reply:
done